### PR TITLE
add controller-google branch for google

### PR DESCRIPTION
**Pull request by HammamAbuShehadeh**
#14 add google controller

### CODE REVIEW

**Comment by HammamAbuShehadeh** [comment]
Done Update scheme models files

File: server/server/config/googleConfig.js
Comment (HammamAbuShehadeh) [inline server/server/config/googleConfig.js:1]: Looks good

File: server/server/config/googleConfig.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/****************************ramah */
```
Comment (HammamAbuShehadeh) [inline server/server/config/googleConfig.js:1]: remove unnecessary comment

File: server/server/config/googleConfig.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+const GoogleStrategy = require('passport-google-oauth20').Strategy;
+const mongoose = require('mongoose');
+const UserModel = require("../models/user");
```
Comment (HammamAbuShehadeh) [inline server/server/config/googleConfig.js:4]: good  import

File: server/server/config/googleConfig.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  }));
+
+  passport.serializeUser((user, done) => {
+    done(null, user.id); 
+    });
+    
+    passport.deserializeUser(function(id, done) {
+    UserModel.findById(id ,(err, user) => done(err, user));
+  });
+};
```
Comment (HammamAbuShehadeh) [inline server/server/config/googleConfig.js:45]: great export to use google config

File: server/server/config/googleConfig.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+/***************************************** */
```
Comment (HammamAbuShehadeh) [inline server/server/config/googleConfig.js:47]: remove unnecessary comment

**Review by HammamAbuShehadeh — COMMENTED** [review]
looks good thx

File: server/server/config/googleConfig.js
Comment (ramah-madi) [reply server/server/config/googleConfig.js:1]: Thank you!

File: server/server/config/googleConfig.js (hunk 1 above)
Comment (ramah-madi) [reply server/server/config/googleConfig.js:1]: Oh ok, I will thx :)

File: server/server/config/googleConfig.js (hunk 2 above)
Comment (ramah-madi) [reply server/server/config/googleConfig.js:4]: Thank you!

File: server/server/config/googleConfig.js (hunk 3 above)
Comment (ramah-madi) [reply server/server/config/googleConfig.js:45]: Thank you very much :)

File: server/server/config/googleConfig.js (hunk 4 above)
Comment (ramah-madi) [reply server/server/config/googleConfig.js:47]: Okk ^_^

**Pull request merged**